frontend/src/views/Org/MembersPage/components/OrgIdentityTab/components/IdentitySection/IdentityUniversalAuthForm.tsx

```diff
@@ -88,7 +88,7 @@ export const IdentityUniversalAuthForm = ({
     } = useForm<FormData>({
         resolver: yupResolver(schema),
         defaultValues: {
-            accessTokenTTL: "2592000",
+            accessTokenTTL: "7200",
             accessTokenMaxTTL: "2592000",
```

Comment:
Changing it back to 30 days so that for basic use cases, things just work and they don't need to immediately think about renew logic